hdf5.rpmlintrc

```diff
@@ -0,0 +1,5 @@
+# This line is mandatory to access the configuration functions
+from Config import *
+
+addFilter('libtool-wrapper-in-package')
+addFilter('binary-or-shlib-defines-rpath')
```

Comment:
Looks like the EL8 build also errors for `static-library-without-debuginfo`.  I didn't see that one as there was no completed EL8 build in the history that I found.

So you will also need:

```suggestion
addFilter('binary-or-shlib-defines-rpath')
addFilter('static-library-without-debuginfo')
```

Reply:
nevermind i didn't know it was still queued and not running

Reply:
ok it's done now and all branches passed.